saleor/graphql/product/types/products.py

```diff
@@ -184,7 +186,14 @@ class ProductVariant(ChannelContextTypeWithMetadata, CountableDjangoObjectType):
         ),
     )
     images = graphene.List(
-        lambda: ProductImage, description="List of images for the product variant."
+        lambda: ProductMedia,
+        description=(
+            "[Deprecated] List of images for the product. Use media field instead. "
```

Comment:
We don't need to change the description, it was correct. To add a deprecation message use the `deprecation_reason` argument - it's a dedicated way of deprecating fields in types.